### ISSUE TITLE
Understanding relation with open.plugncharge-test.hubject.com

### ISSUE THREAD

**Issue by fulup-bzh** [thoughtworks/maeve-csms#20]
I would be please to get some explanation on how/when 'hubject.com' enter in the process and how we may use it to test plug&charge.

Looks like manager/config.toml already have an authorization token. 
* is this token for anyone making test ?
* should every one request a private token ? If then where ?
* what the process to retrieve testing  certificates to feed the ISO15118 ?

**Comment by subnova** [comment]
So Hubject is providing the PKI capabilities for our Plug & Charge functionality. By default MaEVe is configured to use the Hubject test environment https://hubject.stoplight.io/. All the documentation required to use the Hubject test environment is available there. MaEVe automatically retrieves the token needed to make API calls to this environment. This is fine for local testing.

For real-world PnC implementations, you would need to enrol with Hubject at which point you would receive your own API token. You would then provide this to MaEVe using an environment variable: https://github.com/thoughtworks/maeve-csms/blob/main/manager/config/README.md#environment-token-auth-service.

**Comment by fulup-bzh** [comment]
Thank you very much for your quick response. I will look deeper into https://hubject.stoplight.io

We are working on Tux-EVSE a charger open source modular stack in Rust.  Obviously an OCPP backend is required for test. I originally though about using ocpp-rpc, but maeve-csms that I discovered throught a post on Everest mailing list is far more complete and hopefully will allow us to test ISO15118-2/20 with OCPP-201.

Question about  maeve-csms:
* I did not see any graphical dashboard, is there anything ?
* Does it support OCPP-1.6 (white paper extension) for plug&charge with OCPP-1.6
